packages/tree/src/lib/DynamicTreeView.tsx

```diff
@@ -0,0 +1,15 @@
+import { RichTreeView, RichTreeViewProps as DynamicTreeViewProps } from '@mui/x-tree-view/RichTreeView';
+import type { TreeViewBaseItem } from '@mui/x-tree-view/models';
+import { TreeItem } from './TreeItem';
+
+export const DynamicTreeView = ({
```

Comment:
The name is pretty self-explanatory, but we should probably post descriptions for this and treeview just to drive the point.

Reply:
ok, I updated the stories with some descriptions as well.